.github/workflows/check-sdist.yml

```diff
@@ -19,13 +19,13 @@ jobs:
         # scipy and cython are required to build sdist
         run: |
           python -m pip install --upgrade pip
-          pip install check-sdist
+          pip install check-sdist==0.1.4
```

Comment:
I don't think that we want to pin this version. We for sure want to benefit from future development and bug fixes.

Reply:
Ah okee I understand, thank you for the feedback! I've taken it out :)